### DISCUSSION TITLE
How can I build without transform let to var

### DISCUSSION THREAD

**Discussion by gengjiawen** [preactjs/preact#3614] [Q&A] [answered]
Currently the debug build
```js
var EMPTY_OBJ = {};
var EMPTY_ARR = [];
var IS_NON_DIMENSIONAL = /acit|ex(?:s|g|n|p|$)|rph|grid|ows|mnc|ntw|ine[ch]|zoo|^ord|itera/i;
...
```

I tried to change `babel.config.js` target, but no luck.

**Comment by marvinhagemeister** [comment, marked as answer]
What do you mean with "debug build"? Afaik we don't distribute such a thing.

**Reply by rschristian** [reply]
I imagine it's a misunderstanding in what the [`build:debug` script](https://github.com/preactjs/preact/blob/dbe0fa8b1f18b21cd56f85bf1b6d09f968a4427d/package.json#L107) actually does?

For OP, this doesn't create a debug build of `preact`, rather it builds the `debug` subpath (`preact/debug`).

**Reply by gengjiawen** [reply]
> What do you mean with "debug build"? Afaik we don't distribute such a thing.

I mean the non-minify version. `microbundle watch --no-compress`

**Reply by developit** [reply]
`microbundle watch --no-compress --format es`

Also the `main` branch builds ESNext output by default, including in watch mode.

**Comment by developit** [comment]
What's the reason for wanting to skip this transform? We transform let to var because the var version is both smaller and faster.

**Reply by gengjiawen** [reply]
I just curious is there a way to change this. I used to read es6 code.
